Model/Checkout/PlaceOrder.php

```diff
@@ -123,6 +124,7 @@ public function __construct(
      * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
      */
     public function execute(
+        mixed $addressData,
```

Comment:
I believe strict type hinting for `mixed` is only supported from php 8.0. Since there are clients on older versions we should consider leaving this type going out
```suggestion
        $addressData,
```

Reply:
if we use the `$addressData` direct this will happen ` "message": "Class \"addressData\" does not exist",`.